src/test/scala/chiselTests/Assert.scala

```diff
@@ -84,6 +84,78 @@ class PrintableAssumeTester extends Module {
   out := in
 }
 
+class PrintableScopeTester extends Module {
+  val in = IO(Input(UInt(8.W)))
+  val out = IO(Output(UInt(8.W)))
+  out := in
+
+  val w = Wire(UInt(8.W))
+  w := 255.U
+
+  val printableWire = cf"$w"
+  val printablePort = cf"$in"
+}
+
+class AssertPrintableWireScope extends BasicTester {
+  val mod = Module(new PrintableScopeTester)
+  assert(1.U === 2.U, mod.printableWire)
+  stop()
+}
+
+class AssertPrintablePortScope extends BasicTester {
+  ChiselStage.elaborate {
+    new Module {
+      val mod = Module(new PrintableScopeTester)
+      assert(1.U === 2.U, mod.printablePort)
+    }
+  }
+  stop()
+}
```

Comment:
```suggestion
class AssertPrintablePortScope extends BasicTester {
  val mod = Module(new PrintableScopeTester)
  assert(1.U === 2.U, mod.printablePort)
  stop()
}
```
`BasicTester` **is** a Module, it's intended to be a Chisel TestBench testing Chisel inside of it. If you run `elaborate` in side of it, you're doing a separate Chisel elaboration that throws away whatever you built. Thus these tests aren't testing what you think they are unless you just actually instantiate the stuff you want normally as I've shown in this suggestion.

Please fix the tests below as well.